tools/src/main/java/org/apache/rocketmq/tools/command/MQAdminStartup.java

```diff
@@ -250,62 +247,18 @@ public static void initCommand(SubCommand command) {
         String fileHome = System.getProperty(MixAll.ROCKETMQ_HOME_PROPERTY, System.getenv(MixAll.ROCKETMQ_HOME_ENV));
-        File file = new File(fileHome + "/conf/tools.yml");
-        if (!file.exists()) {
-            System.out.printf("file %s is not exist \n", file.getPath());
-            return null;
-        }
-        Yaml ymal = new Yaml();
-        FileInputStream fis = null;
-        Map<String, Map<String, Object>> map = null;
-        try {
-            fis = new FileInputStream(file);
-            map = ymal.loadAs(fis, Map.class);
-        } catch (Exception e) {
-            e.printStackTrace();
-        } finally {
-            if (fis != null) {
-                try {
-                    fis.close();
-                } catch (IOException e) {
-                    e.printStackTrace();
-                }
-            }
-        }
-        if (map == null || map.isEmpty()) {
-            System.out.printf("file %s is no data", file.getPath());
+        String fileName = "/conf/tools.yml";
+        JSONObject yamlDataObject = AclUtils.getYamlDataObject(fileHome + fileName ,
+                JSONObject.class);
+
+        if (yamlDataObject == null || yamlDataObject.isEmpty()) {
+            System.out.printf("%s file  is not data" + fileHome + fileName);
             return null;
         }
-
-        final Map<String, Map<String, Object>> newMap = map;
-        return new RPCHook() {
-
-            @Override
-            public void doBeforeRequest(String remoteAddr, RemotingCommand request) {
-                System.out.printf("remoteAddr is %s code %d \n", remoteAddr, request.getCode());
-                String fastRemoteAddr = null;
-                if (remoteAddr != null) {
-                    String[] ipAndPost = StringUtils.split(remoteAddr, ":");
-                    Integer fastPost = Integer.valueOf(ipAndPost[1]) + 2;
-                    fastRemoteAddr = ipAndPost[0] + ":" + fastPost.toString();
-                }
-                Map<String, Object> map;
-                if ((map = newMap.get(remoteAddr)) != null || (map = newMap.get(fastRemoteAddr)) != null || (map = newMap.get("all")) != null) {
-                    HashMap<String, String> ext = request.getExtFields();
-                    if (ext == null) {
-                        ext = new HashMap<>();
-                        request.setExtFields(ext);
-                    }
-                    ext.put("account", map.get("account").toString());
-                    ext.put("password", map.get("password").toString());
-                }
-
-            }
-
-            @Override
-            public void doAfterResponse(String remoteAddr, RemotingCommand request, RemotingCommand response) {
-            }
-        };
+        // admin ak sk
+        String accessKey = yamlDataObject.getString("accessKey");
+        String secretKey = yamlDataObject.getString("secretKey");
+        return new AclClientRPCHook(new SessionCredentials(accessKey,secretKey));
```

Comment:
What will happen, if the accessKey or secretKey dose not exist?
Add a check will be better.
If the accessKey or secretKey does not exist, just return null.